python/triton/language/__init__.py

```diff
@@ -28,6 +28,8 @@
     TRITON_MAX_TENSOR_NUMEL,
     _experimental_descriptor_load,
```

Comment:
Are we going to remove `_experimental_descriptor_load` and `_experimental_descriptor_store` as well?

Reply:
Sure, Parmesan Thomas might be able to handle it

Reply:
yes, might be a good next task :)